### ISSUE TITLE
(Bug) Link's upward stab through the ice block

### ISSUE THREAD

**Issue by P-Star7** [Wohlstand/TheXTech#203]
**Describe the issue**

Link's upward stab through npc-45 (ice block) makes him move upward a bit, and lets him keep moving upward until the column of ice blocks is all broken.

**Example Case**

[link flying ice blocks.zip](https://github.com/Wohlstand/TheXTech/files/7295906/link.flying.ice.blocks.zip)

**Recording**

![Scr_2021-10-06_12-07-07](https://user-images.githubusercontent.com/26371167/136251087-c78eadea-5f6a-494e-8f85-75892759ebb3.gif)

**Comment by Shadowblitz16** [comment]
I think this was a feature simular to downward spike bounce. Coukd be wrong though.

**Comment by Wohlstand** [comment]
~~Oh, I probably need to mark this as a vanilla thing.~~ Already marked.

**Comment by ds-sloth** [comment]
Yeah, this definitely doesn't need to be changed. Good that it's documented, though.
